### PR TITLE
First commit created all pages and added links

### PR DESCRIPTION
**Pull request by sashaboi**
Yet to do : responsiveness

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    background-image: url("https://wallpapercave.com/wp/wp7698391.jpg");
+    background-repeat: no-repeat;
+    background-size:cover;
+    background-color: rgba(59, 0, 114,1);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding-top: 10rem;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```
Comment (raieegulhane) [inline style.css:12]: Use variables for fonts and color

File: style.css [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+}
+
+.hrclass{
+    background-color: white;
+    border: 1px solid white;margin-top: 1rem;
+    padding-top: 0.01rem;
+    display: flex;
+    flex-direction: column;
+    width: 50rem;
```
Comment (raieegulhane) [inline style.css]: for larger width better to use % than rem

**Review by raieegulhane — COMMENTED** [review]
UI looks nice
Home page is glitching a bit
image

**Review by arulk8 — COMMENTED** [review]
Good to go

**Comment by SuchitaGhadge** [comment]
footer overlapping with start button.

**Comment by sashaboi** [comment]
> 

Looks bad on the screens other than mine, thanks for this.

**Pull request merged**